fastify.d.ts

```diff
@@ -437,7 +442,10 @@ declare namespace fastify {
     /**
      * Add a content type parser
      */
-    addContentTypeParser(contentType: string, opts: {parseAs?: string, bodyLimit?: number} | AsyncContentTypeParser<HttpRequest> | ContentTypeParser<HttpRequest>, parser?: AsyncContentTypeParser<HttpRequest> | ContentTypeParser<HttpRequest>): void;
+    addContentTypeParser(contentType: string, opts: { bodyLimit?: number }, parser: ContentTypeParser<HttpRequest>): void
```

Comment:
The array parameter is supported everywhere in `addContentTypeParser`

Reply:
Fixed